MAINTAINERS.yml

```diff
@@ -2987,6 +2987,23 @@ Networking:
   tests:
     - sample.net.ptp
 
+"Networking: Prometheus":
+  status: maintained
+  maintainers:
+    - mustafaabdullahk
```

Comment:
this is not going to work, because you are not part of the project and did not go through the process described here: https://docs.zephyrproject.org/latest/project/project_roles.html

So I suggest to remove yourself as maintainer, maybe drop this entry completely for now and add it later with yourself as maintainer after gone through the above process.

Reply:
Removed to your suggestion. I will write maybe after gone through the above process.

Reply:
You should just drop the last commit :)

Reply:
A tiny misunderstanding @nashif  :)